packages/snaps-sdk/src/types/methods/notify.ts

```diff
@@ -17,14 +33,47 @@ export enum NotificationType {
  * @property type - The type of notification to display.
  * @property message - The message to display in the notification.
  */
-export type NotifyParams = {
-  type: EnumToUnion<NotificationType>;
-  message: string;
-};
+export type NotifyParams =
+  | {
+      type: EnumToUnion<NotificationType.Native>;
+      message: string;
+    }
+  | {
+      type: EnumToUnion<NotificationType.InApp>;
+      message: string;
+    }
+  | {
+      type: EnumToUnion<NotificationType.InApp>;
+      message: string;
+      detailedView: NotificationComponent;
```

Comment:
What do you think about naming this `content` or something?

Reply:
Yeah, I'll change it.